src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList.tsx

```diff
@@ -36,10 +36,12 @@ const TaskList = ({ tasks, currentWbsNumber, defaultClosed, team, hasTaskPermiss
     setValue(newValue);
   };
 
+  const disableAddTask: boolean = auth.user?.role === 'GUEST' || !team;
```

Comment:
this is incorrect logic... there was a giant thread about permissions in software product requests I think, Ill slack you if you cant find the exact logic. Its pretty similar to the edit task permissions in the task modal, but its not the exact same

Reply:
^